src/libponyc/codegen/genexe.c

```diff
@@ -277,6 +278,9 @@ static bool link_exe(compile_t* c, ast_t* program,
 #ifdef PONY_USE_LTO
     "-flto -fuse-linker-plugin "
 #endif
+#if defined(PLATFORM_IS_ALPINE_LINUX) || defined(PLATFORM_IS_FREEBSD)
```

Comment:
@Shizmob I agree that best solution is to do a test and then base a decision on that however that was going to add a level of complexity into the ponyc linking code that I wasn't comfortable with. If others agree that's the route desired, then we can implement the appropriate solution.

Can you give me an example set of steps/commands to reproduce where it breaks cross-compilation?